packages/neos-ui/src/styleHostOnly.css

```diff
@@ -1,25 +1,81 @@
 @import 'normalize.css';
 
 @font-face {
-  font-family: 'Work Sans';
-  src: url('../../../Resources/Public/Fonts/WorkSans-Regular.woff') format('woff');
-  font-weight: normal;
-  font-style: normal;
+    font-family: 'Open Sans';
+    src: url('../../../Resources/Private/Fonts/OpenSans-Light.woff');
+    font-weight: 300;
+    font-style: normal;
 }
 
 @font-face {
-  font-family: 'Work Sans';
-  src: url('../../../Resources/Public/Fonts/WorkSans-Bold.woff') format('woff');
-  font-weight: bold;
-  font-style: normal;
+    font-family: 'Open Sans';
```

Comment:
Do we need all of those font weights? That would be like tons of data...

Reply:
@dimaip Based on the discussion on the UX Guild, I think we need more weights in the future. Currently we use only 2 weight (regular and bold), so only used weights are loaded.

Before this change the font is base64 encoded in the CSS, so the CSS was around 500Ko, now we have a really slim CSS, and even if all weights are loaded we save bandwidth. Fonts can be cached (http, or localstorage in the future, if we need to improve this)

Reply:
@dfeyer hey, but let's add font weights only when we need them, when the design changes? We ship something when it's not used?
Currently we don't base64 fonts afaik, @mstruebing fixed it a while ago.

Reply:
@dfeyer do you agree?